dotnet/ComputerVision/ExtractText/Program.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.CognitiveServices.Samples.ComputerVision.ExtractText
+{
+    using Newtonsoft.Json.Linq;
+
+    public class Program
+    {
+        public const string subscriptionKey = "<your training key here>"; //Insert your Cognitive Services subscription key here
+        public const string endpoint = "https://westus.api.cognitive.microsoft.com"; // You must use the same Azure region that you generated your subscription keys for.  Free trial subscription keys are generated in the westus region. 
+
+        static void Main(string[] args)
+        {
+            ExtractTextSample.RunAsync(endpoint, subscriptionKey).Wait(5000);
+
+            Console.WriteLine("\nPress ENTER to exit.");
+            Console.ReadLine();
+        }
+    }
+
+    public class ExtractTextSample
+    {
+        public static async Task RunAsync(string endpoint, string key)
+        {
+            Console.WriteLine("Extracting text from the images:");
+
+            string imageFilePath = @"Images\handwritten_text.jpg"; // See this repo's readme.md for info on how to get these images. Alternatively, you can just set the path to any appropriate image on your machine.
+            string remoteImageUrl = "https://github.com/Azure-Samples/cognitive-services-sample-data-files/raw/master/ComputerVision/Images/printed_text.jpg";
+
+            await ExtractTextFromStreamAsync(imageFilePath, endpoint, key, "Handwritten");  //the last parameter is whether the text that has to be extracted is printed or handwritten 
+            await ExtractTextFromUrlAsync(remoteImageUrl, endpoint, key, "Printed"); //This textRecognitionMode can only be either Handwritten or Printed
+        }
+
+        
+        static async Task ExtractTextFromStreamAsync(string imageFilePath, string endpoint, string subscriptionKey, string mode="Handwritten")
```

Comment:
It will be better to make the mode a required parameter. Having it default to a particular mode for Stream and another for the Url case is strange. It might be better to also rename it to textRecognitionMode, so it matches the way you are referencing it in the comments.